test_util/marathon.py

```diff
@@ -237,7 +237,7 @@ def _wait_for_pod_deployment(pod_id):
             r = self.get('v2/deployments')
             data = r.json()
             if len(data) > 0:
-                logging.info('Waiting for pod to be deployed %s', repr(data))
+                logging.info('Waiting for pod to be deployed %r', data)
```

Comment:
If we are going for more idiomatic, I would do 'Waiting for pod to be deployed {}'.format(data)